src/EventProcessor/CustomerEventProcessor.php

```diff
@@ -24,12 +21,12 @@ public function __construct(protected RobinClient $robinClient)
     }
 
     /**
-     * @param EventInterface|CustomerEvent $event
```

Comment:
Thinks we need to keep this. This typehint would be here for IDE autocompletion, as not every implementation of `EventInterface` has `getCustomer` method.

Reply:
Reverted reference to concrete event class.